index.html

```diff
@@ -48,112 +54,156 @@ <h2>I’am a Software Developer</h2>
-        <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-        </p>
-        <ul>
-            <li class="html2">html</li>
-            <li class="bootstrap2">bootstrap</li>
-            <li class="Ruby2">ruby</li>
-        </ul>        
-    </div>
-    <button class="second-btn" type="button">See Project</button>
-    <div class="profession">
-        <h4 class="pro">Profesional Art Printing Data</h4>
-        <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-        </p>
-        <ul>
-            <li class="html2">html</li>
-            <li class="bootstrap2">bootstrap</li>
-            <li class="Ruby2">ruby</li>
-        </ul>        
-    </div>
-    <button class="second-btn" type="button">See Project</button>
-    <div class="profession">
-        <h4 class="pro">Profesional Art Printing Data</h4>
-        <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-        </p>
-        <ul>
-            <li class="html2">html</li>
-            <li class="bootstrap2">bootstrap</li>
-            <li class="Ruby2">ruby</li>
-        </ul>        
-    </div>
-    <button class="second-btn" type="button">See Project</button>    
-    <div class="profession">
-        <h4 class="pro">Profesional Art Printing Data</h4>
-        <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-        </p>
-        <ul>
-            <li class="html2">html</li>
-            <li class="bootstrap2">bootstrap</li>
-            <li class="Ruby2">ruby</li>
-        </ul>        
-    </div>
-    <button class="second-btn" type="button">See Project</button>
-    <div class="profession">
-        <h4 class="pro">Profesional Art Printing Data</h4>
-        <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-        </p>
-        <ul>
-            <li class="html2">html</li>
-            <li class="bootstrap2">bootstrap</li>
-            <li class="Ruby2">ruby</li>
-        </ul>        
-    </div>
-    <button class="second-btn" type="button">See Project</button>
-    <div class="profession">
-        <h4 class="pro">Profesional Art Printing Data</h4>
-        <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-        </p>
-        <ul>
-            <li class="html2">html</li>
-            <li class="bootstrap2">bootstrap</li>
-            <li class="Ruby2">ruby</li>
-        </ul>        
+    <div class="works">
+        <div class="recent">
+            <div class="title">
+                <h2 class="first">My Recent Works</h2>
```

Comment:
- [ ] kindly increase the size of this `h2` to match figma design 

**_your design :_**
![image](https://user-images.githubusercontent.com/89282221/188262591-b1d200a3-6b2c-4964-afb5-8e719533b900.png)


**_figma design :_**
![image](https://user-images.githubusercontent.com/89282221/188262602-1b51feab-8888-474c-8183-da5160a88c22.png)